### ISSUE TITLE
Unable to close Paypal from ionic application 

### ISSUE THREAD

**Issue by yashp98** [Enngage/ngx-paypal#165]
I have ionic application. I am using this package in ionic application . but when I open PayPal popup in small device close button is hide .unable to go back to app and for notch device close button is cut down .any one know how to open in browser or any other solution.
![Simulator Screen Shot - iPod touch (7th generation) - 2021-02-26 at 09 46 06](https://user-images.githubusercontent.com/35022242/109254547-3e9da880-7818-11eb-9055-77ca2f5edbd8.png)
 
<img width="534" alt="image" src="https://user-images.githubusercontent.com/35022242/109254774-b075f200-7818-11eb-95e9-60b54ca6e4c1.png">

**Comment by marshall86** [comment]
facing the same issue.. would be nice to have a way to customize that overlay as well, to remove the white border i.e.

**Comment by marshall86** [comment]
I managed to remove the ugly white border and the close button.

You need to put this style in your custom.scss file or any other global css file:

```
html.ios .paypal-checkout-sandbox-iframe {
    top: 25px !important; 
    border: 0;
}

```

this style will remove the white border if you like and move the overlay slightly down, you can really style the close button because it's inside an iframe that is generated only once the paypal js sdk is loaded.

**Comment by marshall86** [comment]
`
.paypal-checkout-sandbox-iframe {
    border: 0;
}

html.plt-ios .paypal-checkout-sandbox-iframe {
    top: env(safe-area-inset-top) !important;
}
`

This is the best way
